Rubberduck.Core/UI/CodeExplorer/Commands/AddTemplateCommand.cs

```diff
@@ -59,6 +66,25 @@ private bool SpecialEvaluateCanExecute(object parameter)
             }
         }
 
+        private bool EvaluateCanExecute(ICodeExplorerNode node)
+        {
+            if (!(node is CodeExplorerItemViewModel)
+                || node.Declaration == null)
+            {
+                return false;
+            }
+
+            try
+            {
+                var project = node.Declaration.Project;
+                return AllowableProjectTypes.Contains(project.Type);
+            }
+            catch (COMException)
+            {
+                return false;
```

Comment:
I'm inclined to think that it should at least get logged to `Debug` or `Trace` rather than silently swallowing.

Reply:
Will log ad `Debug`.

Reply:
I would like to revert my position. The entire `try` should go. Handling exceptions by logging them is already done in `CommandBase`.